controllers/borrowerController.js

```diff
@@ -202,9 +202,9 @@ router.get('/branches', async (req, res) => {
 });
 
 // Read all book copies in a specific branch
-router.get('/branches/:id/copies', async (req, res) => {
+router.get('/branches/:id/copies/skip/:skip/limit/:limit', async (req, res) => {
```

Comment:
These should be query params instead. They don't define our query they refine it.